Utilities/DataLoading/EdFi.XmlLookup.Console/Program.cs

```diff
@@ -55,32 +46,45 @@ private static void Main(string[] args)
             {
                 try
                 {
-                    using (var container = new Container())
-                    {
-                        LogConfiguration(p.Object);
+                    var container = RegisterContainer();
+                    using var scope = container.BeginLifetimeScope();
 
-                        // configure DI container
-                        ConfigureCompositionRoot(container, p.Object);
+                    LogConfiguration(p.Object);
 
-                        // retrieve application
-                        var application = container.GetInstance<XmlLookupApplication>();
+                    // configure DI container
 
-                        // run application 
-                        exitCode = application.Run().Result;
-                    }
+                    // retrieve application
+                    var application = container.Resolve<XmlLookupApplication>();
+
+                    // run application
+                    exitCode = application.Run().Result;
```

Comment:
You're changing the other consoles to be async, why not fix this one too?

Reply:
done